db/migrate/20220219192150_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Text
+      t.date :CreatedAt
+      t.date :UpdatedAt
```

Comment:
- [ ] posts table is miss the `title` column. Kindly add it. A post should have a title and text.